src/ethereum/rainbowkit/RainbowkitConfig.tsx

```diff
@@ -81,7 +79,7 @@ function RainbowkitConfig({
 
   return (
     <WagmiConfig client={client}>
-      <RainbowKitProvider chains={filteredChains} initialChain={initialChain}>
```

Comment:
Is this no longer required. i.e what happens if you are not connected and the hash is Goerli for instance? What chain will it connect to?